docs/conf.py

```diff
@@ -1 +1,2 @@
 from crate.theme.rtd.conf.crate_python import *
+html_favicon = None
```

Comment:
what does that symbol mean? 
![image](https://user-images.githubusercontent.com/158125/27393416-8b8bd494-56a9-11e7-8c5e-d8f404e5ae5b.png)